ci/checks/copyright.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020-2022, NVIDIA CORPORATION.
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.
```

Comment:
This file was updated to better support `pre-commit` (I copied these changes from cudf). It also supports autofixing now, so if the style checks fail, it will tell you exactly which files need to change and what they should say.